tests/functional/storageclass/test_ec_default_pool_and_sc.py

```
@@ -0,0 +1,262 @@
+        # Step 4: Create PVC without storageClassName
+        pvc_data = templating.load_yaml(constants.CSI_PVC_YAML)
+        pvc_data["metadata"]["name"] = helpers.create_unique_resource_name(
+            "ec-default", "pvc"
+        )
+        log.info("Creating PVC without storageClassName to test default SC binding")
+        del pvc_data["spec"]["storageClassName"]
+        pvc_obj = pvc_factory(custom_data=pvc_data, status=constants.STATUS_BOUND)
+        assert pvc_obj, "PVC creation failed"
+
+        bound_sc = pvc_obj.get().get("spec").get("storageClassName")
+        log.info(f"PVC bound to StorageClass: {bound_sc}")
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)